src/agents/common/state.py

```diff
@@ -129,6 +129,8 @@ def get_messages_including_summary(self) -> list[MessageLikeRepresentation]:
 
     def all_tasks_completed(self) -> bool:
```

Comment:
it seems this method is not used anywhere. Can you remove it?

Reply:
Even better. Done.